src/helm/benchmark/scenarios/lsat_qa_scenario.py

```diff
@@ -102,8 +102,10 @@ def __init__(self, task):
     def get_question_types(self, tags: List[str]) -> List[str]:
         question_type: str = tags[2].replace("grouping (distribution)", "distribution grouping") or "miscellaneous"
         main_type = self.subtype2type.get(question_type)
-        assert main_type
-        return [question_type.replace(" ", "_").replace("/", "_"), main_type]
+        if main_type is not None:
+            return [question_type.replace(" ", "_").replace("/", "_"), main_type]
+        else:
+            return [main_type]
```

Comment:
Here with the `if main_type is not None` you are basically return `[None]`. Shouldn't this be `return [question_type.replace(" ", "_").replace("/", "_")]` instead?

Reply:
Oops, fixed.